src/go/wsl-helper/cmd/kubeconfig.go

```diff
@@ -46,8 +46,21 @@ var kubeconfigCmd = &cobra.Command{
 	Long:  `This command configures the Kubernetes configuration inside a WSL2 distribution.`,
 	Args:  cobra.ExactArgs(0),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+
 		configPath := kubeconfigViper.GetString("kubeconfig")
 		enable := kubeconfigViper.GetBool("enable")
+		verify := kubeconfigViper.GetBool("verify")
+
+		configDir := path.Join(homedir.HomeDir(), ".kube")
+		linkPath := path.Join(configDir, "config")
+		unsupportedConfig, symlinkErr := requireManualSymlink(linkPath)
+		if verify {
+			if unsupportedConfig {
```

Comment:
Shouldn't we also fail if `symlinkErr` is not `nil`?

Reply:
That's a concern for when we are manipulating the actual file, e.g. when we attempt to remove it which takes place later. The `verify` flag should care about whether the contents of the kubeConfig are rancher desktop only or not.

Reply:
Alright, then.